WooCommerce/src/main/kotlin/com/woocommerce/android/ui/bookings/list/BookingListScreen.kt

```diff
@@ -254,6 +248,32 @@ private fun BookingList(
     }
 }
 
+@Composable
+private fun BookingListIsLoading(controlsState: BookingListControlsState) {
```

Comment:
⛏️ Nitpick, but the `is` in the name was a bit confusing, and I kinda expected two states for loading and not. Probably because that's a common prefix for Boolean values. 
```suggestion
private fun BookingListLoading(controlsState: BookingListControlsState) {
```

Reply:
I totally agree. Done: 8b65ac3b974e198dc434cebe2f38f513739b4fda